.goreleaser.yml

```
@@ -23,7 +23,7 @@ builds:
       - -s
       - -w
       - -X github.com/hetznercloud/hcloud-cloud-controller-manager/hcloud.providerVersion={{ if not .IsSnapshot }}v{{ end }}{{ .Version }}
-      - -X k8s.io/component-base/version.gitVersion={{ if not .IsSnapshot }}v{{ end }}{{ .Version }}
+      - -X k8s.io/component-base/version.gitVersion=v0.0.0-master+{{ if not .IsSnapshot }}v{{ end }}{{ .Version }}
```

```suggestion
      - -X k8s.io/component-base/version.gitVersion=v0.0.0-master+v{{ .Version }}
```

@apricote Do you think we still need the `{{ if not .IsSnapshot }}v{{ end }}`?

---

Nah, as the new version is just inside the "build metadata" (in Semver lingo) the v prefix is not required anymore and you can remove it.